### PR TITLE
Features

### PR DESCRIPTION
**Pull request by edwinmoradian90**
So we did the project a little differently. We cloned the whole website, but it contains all the main features.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,20 @@
 # design-teardown
-microverse project: design teardown
+
+This is the fifth project of the Main HTML/CSS curriculum at [Microverse](https://www.microverse.org/) - @microverseinc
+* The goal was to recreate Smashing Magazine's main page 
+* The project was completed using html and css 
+
+#### [Assignment link](https://www.theodinproject.com/courses/html5-and-css3/lessons/design-teardown)
+
```
Comment (stratospherique) [inline README.md]: The live version is not working. I guess because you used the html file of the master branch

File: index.html [hunk 2]
```diff
@@ -1,13 +1,596 @@
...
+      <li><a href="#" class="footer_link">Workflow</a></li>
+    </ul>
+
+    <p>With a commitment to quality content for the design community.</p>
+    <p>Founded by Vitaly Friedman and Sven Lennartz. 2006–2019.</p>
+    <p>Smashing is proudly running on <a href="#">Netlify</a>.</p>
+    <p>Fonts by <a href="#">Latinotype</a>.</p>
+
+    <div>
+      <figure>
```
Comment (stratospherique) [inline index.html]: - This the footer part that needs to be centered

**Comment by edwinmoradian90** [comment]
Thanks for feedback. On HD resolution, we don't see any overflowing. We intended the site to look good in HD and didn't plan on making it responsive since we are not in the responsive section yet. We fixed the bird picture

**Comment by stratospherique** [comment]
> Thanks for feedback. On HD resolution, we don't see any overflowing. We intended the site to look good in HD and didn't plan on making it responsive since we are not in the responsive section yet. We fixed the bird picture

### Splendid work :clap: :clap:. Kindly submit another code review request. Just to for final checks and in order to give you our approval to move to the next project :muscle: .

**Comment by samgaco** [comment]
#### Hi @flov3rh and @edwinmoradian90 :vulcan_salute:  Since everything is covered you are approved :+1: You can now merge to the master branch and submit the project completion form. Congratulations :tada:

**Comment by edwinmoradian90** [comment]
awesome

**Pull request merged**